12_glob_lab/globlab.txt

```diff
@@ -1,29 +1,59 @@
 Glob Lab.
 https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
-1. Type in a glob pattern to match:
+1. 
+
+Type in a glob pattern to match:
 
-  1. All files whose name starts with a lowercase letter
+ 
+ 1. All files whose name starts with a lowercase letter
   Answer: 
 
-  2. All files whose name starts with an uppercase letter
+ 
+ls -d [[:lower:]]*/*
+ 2. All files whose name starts with an uppercase letter
   Answer:
 
-  3. All files whose name ends in an "a", "e", "i", "o" or "u"
+ 
+ls -d [[:upper:]]*/* - for some reason shows the following:
+Doc/README.txt Doc/test.c
+Why test.c as well?

```

Comment:
Can you please explain why

>  Or, if you also want to see the contents of each directory drop the d:
> 
> ls [[:upper:]]*
shows me the following? why also the lower case file
![glob](https://cloud.githubusercontent.com/assets/28634107/26285619/2beca708-3e5c-11e7-93bd-4bf0a7c0a991.JPG)